jenkins/ci.suse.de/ardana-job.yaml

```diff
@@ -177,14 +177,19 @@
 
           cat ardana_net_vars.yml
 
-          ansible-playbook -vvv -i hosts ssh-keys.yml
-          ansible-playbook -vvv -i hosts -e "build_url=$BUILD_URL" \
-                                         -e "cloudsource=${cloudsource}" repositories.yml
-          ansible-playbook -vvv -i hosts -e "deployer_floating_ip=$DEPLOYER_IP" \
-                                         -e "deployer_model=${model}" \
-                                         -e "tempest_run_filter=${tempest_run_filter}" \
-                                         init.yml
-
+          ansible-playbook -v -i hosts ssh-keys.yml
+          ansible-playbook -v -i hosts -e "build_url=$BUILD_URL" \
+                                       -e "cloudsource=${cloudsource}" repositories.yml
+          ansible-playbook -v -i hosts -e "deployer_floating_ip=$DEPLOYER_IP" \
+                                       -e "deployer_model=${model}" \
+                                       init.yml
+          # Run site.yml outside ansible for output streaming
+          ssh  -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no ardana@$DEPLOYER_IP \
+              "cd ~/scratch/ansible/next/ardana/ansible ; \
+               ansible-playbook -vvv -i hosts/verb_hosts site.yml"
```

Comment:
@aspiers if site.yml fails the job is aborted, and post-deploy-checks are not invoked. I think thats a reasonable behavior. pre/post deploy might spit errors that are irrelevant otherwise if install doesn't finish.